systemTechniques.ldif

```diff
@@ -0,0 +1,226 @@
+version: 1
```

Comment:
This is a working file. It should not be put in this repository and is not exactly up to date.
But it is way easier for me to put it in the same PR than the technique changes.